packages/tc-ui/src/pages/edit/browser.js

```diff
@@ -21,7 +21,7 @@ const initDocumentEditors = () => {
 const initRelationshipSelectors = entireRecord => {
 	[
 		...document.querySelectorAll(
-			'[data-component="relationship-picker"]:not([data-disabled])',
+			'[data-component="relationship"]:not([data-disabled])',
 		),
 	].forEach(container => attachRelationshipPicker(container, entireRecord));
 };
```

Comment:
No need to spread a nodelist into an array, `NodeList.prototype.forEach` has been a thing for a long time!

Reply:
This is done quite a bit and not very well tested. Don't want to either a) tidy just this one b) tidy them all but introduce a mistake... so gonna leave it for now